packages/terra-base/src/terra-dev-site/doc/base/Base.1.doc.mdx

```diff
@@ -77,3 +85,10 @@ const locale = (navigator.languages && navigator.languages[0])
 * [Localization Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#internationalization-i18n-support)
 * [react_on_rails Compatible](https://github.com/shakacode/react_on_rails/blob/8cb06ed35cb5c2c453bcc193282b4c091574c1b7/docs/basics/i18n.md#how-to-add-i18n)
 * [CND Compatible](https://github.com/webpack/docs/wiki/configuration#outputpublicpath)
+
```

Comment:
Does the AutoGenerated Peer dependency script still work with this change?

Reply:
No, it won't work with this change.